sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -540,6 +540,106 @@ except ResourceNotFoundError:
     print("Successfully deleted model with id {}".format(custom_model.model_id))
 ```
 
+### Convenience methods to get items from a specific model (**Beta feature**)
```

Comment:
Ah I thought when you said README you meant add get_lines() and get_words() to the existing examples. Do you think adding a dedicated sample with this info could work? which we could link to from the readme? We've gotten feedback to keep READMEs brief / focused on the big features so I'm not sure this is the right place for it.
Adding @maririos @kinelski @samvaity @witemple-msft for more perspectives on how they plan to show the feature.

Reply:
Yeah that sounds better, will update.

Reply:
Done, removed from the readme but added a get words in one of the examples.